### DISCUSSION TITLE
NEW PROJECTS: By default status of an "issue" must be "Todo"

### DISCUSSION THREAD

**Discussion by jayaramkasi** [community/community#8922] [Projects and Issues] [answered]
Right now, it is being tagged into "No Status".

**Comment by zaporylie** [comment, marked as answer]
You can go to the project automation menu and enable the workflow that will do just that.

**Reply by jayaramkasi** [reply]
Is it possible in the New Beta Projects? I was talking about that

**Reply by zaporylie** [reply]
Me too. Yes, it is possible.

**Reply by zaporylie** [reply]
![image](https://user-images.githubusercontent.com/1690685/146511923-2df8e346-af37-4af2-bcc4-94cac4147464.png)
Just follow the arrows 😉

**Reply by jayaramkasi** [reply]
Missed it.. Cheers mate! @zaporylie
